README.md

```diff
@@ -1,23 +1,24 @@
-| Launcher API | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.framework.api.launcher&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.framework.api.launcher) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.framework.api.launcher&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.framework.api.launcher) | [![Coverage](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.framework.api.launcher&metric=coverage)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.framework.api.launcher) | 
-| Test Catalog API | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.framework.api.testcatalog&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.framework.api.testcatalog) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.framework.api.testcatalog&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.framework.api.testcatalog) | [![Coverage](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.framework.api.testcatalog&metric=coverage)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.framework.api.testcatalog) | 
-| Utilities Maven Plugin | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Agalasautils-maven-plugin&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Agalasautils-maven-plugin) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Agalasautils-maven-plugin&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Agalasautils-maven-plugin) | [![Coverage](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Agalasautils-maven-plugin&metric=coverage)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Agalasautils-maven-plugin) | 
+This repository contains the code  for Galasa's core test framework. The framework orchestrates all component activities, and co-ordinates with the test runner to execute your tests.
```

Comment:
core framework,   think the test is unnecessary and can be confusing.   Need to strip out references to "Test Runner".   That is a junit term.